homeassistant/components/dsmr/translations/en.json

```diff
@@ -15,14 +15,16 @@
                 "data": {
                     "dsmr_version": "Select DSMR version",
                     "host": "Host",
-                    "port": "Port"
+                    "port": "Port",
+                    "is_rfxtrx": "RFXtrx with integrated P1 reader"
```

Comment:
I'm not sure how to deal with translations.
Is there anything I should do?

Reply:
You need to change `strings.json` instead of this file, the translations are automatically generated once merged. For testing purposes you could edit en.json as well.

Reply:
Thanks for the information. Updated accordingly.